nx-X11/programs/Xserver/include/resource.h

```diff
@@ -58,7 +58,6 @@ SOFTWARE.
 typedef unsigned long RESTYPE;
 
 #define RC_VANILLA	((RESTYPE)0)
-#define RC_CACHED	((RESTYPE)1<<31)
```

Comment:
They might have just forgotten to remove it, then. But given it's still there even in the newest version of xorg-xserver, we should also retain it for a smaller diff.